pkgs/development/python-modules/mkdocs-drawio-exporter/default.nix

```diff
@@ -5,26 +5,33 @@
   isPy3k,
   lib,
   mkdocs,
+  poetry-core,
+  livereload,
+  tornado,
 }:
 
 buildPythonPackage rec {
   pname = "mkdocs-drawio-exporter";
   version = "0.9.1";
-  format = "setuptools";
+  format = "pyproject";
 
   disabled = !isPy3k;
 
   src = fetchPypi {
-    inherit pname version;
+    pname = "mkdocs_drawio_exporter";
+    inherit version;
     hash = "sha256-x8X8hvN/tL8C6VhgMCEHDh2hILjBoyLgQfsFD1+qXgo=";
   };
 
   propagatedBuildInputs = [
+    poetry-core
```

Comment:
This is a build-time dependency, and should go into `build-systems` instead.

Reply:
Dependencies is the new lingo that matches build-system, so feel free to update that.

Reply:
Cool, done!